openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1946,9 +1946,19 @@ message NativeResponse {
     // The URL of the image or js.
     // REQUIRED for image or js, optional for custom.
     optional string url = 3;
+    
+    // Array of key:value objects for custom tracking.
+    // To be agreed individually with the exchange, for example the account number of the DSP with a tracking company.
+    // IE {“accountnumber”:”123”}.
+    repeated KeyValuePair customdata = 4;
 
     // Extensions.
     extensions 100 to 9999;
+    
+    message KeyValuePair {
```

Comment:
This seems a useful object, I expect it would be valuable to put it at the top level of the proto definition for re-use -- do you agree?

Reply:
sounds good. will do as soon as I have a free minute